Strings/Details+Transit.jsonc

```
@@ -138,31 +138,31 @@
 
 	"駅業務機器": {
 		"ja": "駅業務機器",
-		"en": null,
+		"en": "Station Equipment",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"簡易改札機": {
 		"ja": "簡易改札機",
-		"en": null,
+		"en": "Standalone Card Reader",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"携帯端末": {
 		"ja": "携帯端末",
-		"en": null,
+		"en": "Portable Terminal",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"現金/なし": {
 		"ja": "現金/なし",
-		"en": null,
+		"en": "No Funds",
```

こちらは「現金」または「なし」を意図したものでした。
This was intended to be "cash" or "none".
```suggestion
		"en": "Cash / None",
```